src/parameters/all_parameters.cpp

```diff
@@ -58,6 +58,28 @@ void AllParameters::declare_parameters (dealii::ParameterHandler &prm)
                       dealii::Patterns::Bool(),
                       "Use other boundary conditions by default. Otherwise use periodic (for 1d burgers only");
 
+    prm.declare_entry("use_energy", "false",
+                      dealii::Patterns::Bool(),
+                      "Not calculate energy by default. Otherwise, get energy per iteration.");
+
+    prm.declare_entry("use_L2_norm", "false",
+                      dealii::Patterns::Bool(),
+                      "Not calculate L2 norm by default (M+K). Otherwise, get L2 norm per iteration.");
+
+    prm.declare_entry("use_classical_Flux_Reconstruction", "false",
+                      dealii::Patterns::Bool(),
+                      "Not use Classical Flux Reconstruction by default. Otherwise, use Classical Flux Reconstruction.");
+
+    prm.declare_entry("flux_reconstruction", "cDG",
+                      dealii::Patterns::Selection("cDG | cSD | cHU | cNegative | cNegative2 | cPlus | cPlus1D |c10Thousand | cHULumped"),
+                      "Flux Reconstruction. "
+                      "Choices are <cDG | cSD | cHU | cNegative | cNegative2 | cPlus | cPlus1D | c10Thousand | cHULumped>.");
+
+    prm.declare_entry("flux_reconstruction_aux", "kDG",
+                      dealii::Patterns::Selection("kDG | kSD | kHU | kNegative | kNegative2 | kPlus | k10Thousand"),
+                      "Flux Reconstruction for Auxiliary Equation. "
+                      "Choices are <kDG | kSD | kHU | kNegative | kNegative2 | kPlus | k10Thousand>.");
+
```

Comment:
I'm guessing some of those refer to some of your test parameters? Like whether you calculate the energy or l2 norm within the test? You might want to put those in a different subsection. We can merge this pull request an open an issue assigned to you to be fixed later? How does that sound?

Reply:
Yes these are all parameters that will be used/properly defined in merge part 2. Basically energy = M+K norm, l2 = M norm, classical flux reconstruction is only reconstructing the surface integral, then the parameters correspond to all the different FR schemes. 
Sounds good leaving it as an open issue, it will be fixed in merge part 2!